app/app.go

```diff
@@ -631,6 +631,13 @@ func callValidatorMock(ctx context.Context, duty core.Duty, cl eth2client.Servic
 		} else {
 			log.Info(ctx, "Mock block proposal submitted to validatorapi", z.I64("slot", duty.Slot))
 		}
+	case core.DutyBuilderProposer:
+		err := validatormock.ProposeBlindedBlock(ctx, cl.(*eth2http.Service), signer, eth2p0.Slot(duty.Slot), addr, pubshares...)
```

Comment:
can assign `eth2Svc := cl.(*eth2http.Service)` since the typecast happens in all cases

Reply:
cool done